README.md

```diff
@@ -80,6 +96,29 @@ Ran 2 tests in 0.033s
 OK
 ```
 
+
+    
+# Setup pre-commit hook to check for secrets
+
+We use [pre-commit](https://pre-commit.com/) to setup pre-commit hooks for this repo.
+We use [detect-secrets](https://github.com/Yelp/detect-secrets) to search for secrets being committed into the repo. 
+
+To install the pre-commit hook, run
+```
+pre-commit install
+```
+
+To update the .secrets.baseline file run
+```
+detect-secrets scan --update .secrets.baseline
+git add .secrets.baseline
```

Comment:
Don't need the `git add` line as this commit is already adding the baseline file.